handlers/apps.go

```diff
@@ -0,0 +1,77 @@
+package handlers
+
+
+import (
+	"fmt"
+	"github.com/knative-scout/app-api/models"
+	"go.mongodb.org/mongo-driver/bson"
+	"net/http"
+	"strings"
+)
+
+// AppsHandler is used search apps and return result
+// in case of an empty query, it returns all the apps in the database
+
+type AppsHandler struct {
```

Comment:
Can you name this something less general. B/c we will have a few more app endpoints.

Maybe `AppSearchHandler`?

Then rename the `handlers/apps.go` file to something less general like `handlers/search,go`

Reply:
okay